www/test/unit/controller/home.controller.test.js

```diff
@@ -1,27 +1,60 @@
 describe('HomeController', function() {
-  var scope, placeFactory, state;
+  var scope, placeFactory, state, place;
 
   beforeEach(function() {
     module('vamosJuntas');
 
-    inject(function ($rootScope, $controller, $injector, $httpBackend) {
-        scope = $rootScope.$new();
-        placeFactory = $injector.get('placeFactory');
+    inject(function($rootScope, $controller, $injector, $httpBackend) {
+      scope = $rootScope.$new();
+      placeFactory = $injector.get('placeFactory');
 
-        createController = function() {
-          $controller('HomeController', {
-              '$scope': scope,
-              'placeFactory': placeFactory,
-          });
-        };
+      createController = function() {
+        $controller('HomeController', {
+          '$scope': scope,
+          'placeFactory': placeFactory,
+        });
+      };
     });
 
-    spyOn(placeFactory, 'fetchPlaces');
+    place = {
+      "title": "Chafariz da Redenção",
+      "occurrences": [{
+          "risk": "Local Deserto",
+          "date": "10/10/2015",
+          "period":"Manhã",
+          "numberOfOccurrences": 3
+        }, {
+          "risk": "Mal Iluminado",
+          "date": "10/10/2015",
+          "period":"Manhã",
+          "numberOfOccurrences": 6
+        }, {
+          "risk": "roubo",
+          "date": "10/10/2015",
+          "period":"Manhã",
+          "numberOfOccurrences": 6
```

Comment:
We are planning to return `count` instead of `numberOfOccurrences` from the API.